Maths/GetEuclidGCD.js

```diff
@@ -1,32 +1,22 @@
-/*
-    Problem statement and Explanation : https://en.wikipedia.org/wiki/Euclidean_algorithm
-
-    In this method, we have followed the iterative approach to first
-    find a minimum of both numbers and go to the next step.
-*/
-
 /**
- * GetEuclidGCD return the gcd of two numbers using Euclidean algorithm.
- * @param {Number} arg1 first argument for gcd
- * @param {Number} arg2 second argument for gcd
- * @returns return a `gcd` value of both number.
+ * GetEuclidGCD Euclidean algorithm to determine the GCD of two numbers
+ * @param {Number} a integer (may be negative)
+ * @param {Number} b integer (may be negative)
+ * @returns {Number} Greatest Common Divisor gcd(a, b)
  */
-const GetEuclidGCD = (arg1, arg2) => {
-  // firstly, check that input is a number or not.
-  if (typeof arg1 !== 'number' || typeof arg2 !== 'number') {
-    return new TypeError('Argument is not a number.')
+export function GetEuclidGCD (a, b) {
+  if (typeof a !== 'number' || typeof b !== 'number') {
+    throw new TypeError('Arguments must be numbers')
   }
-  // check that the input number is not a negative value.
-  if (arg1 < 1 || arg2 < 1) {
-    return new TypeError('Argument is a negative number.')
+  a = Math.abs(a)
```

Comment:
This is unnecessary, as you are already converting negative numbers to positive in lines 14-15.